packs/spells/cinder-gaze.json

```diff
@@ -20,17 +20,34 @@
             "value": ""
         },
         "damage": {
-            "value": {}
+            "value": {
+                "hC5GKInrfS2NyRMF": {
+                    "applyMod": true,
+                    "type": {
+                        "categories": [],
+                        "subtype": "",
+                        "value": ""
+                    },
+                    "value": "1"
+                }
+            }
```

Comment:
This doesn't deal damage

Reply:
A TempHP rule element.

Reply:
The issue with setting damage on the spell is bonuses to damage that apply to spells, like inspire courage, would affect this too.

Reply:
That makes sense, but I don't know of a way to do "1 + spellcasting ability modifier" in a formula. Does an `@attribute` exist for the value of your spellcasting ability mod?

Reply:
Well, it works, if only moderately hacky